Dockerfile

```diff
@@ -25,18 +30,25 @@ COPY --link Cargo.toml ./
 COPY --link Cargo.lock ./
 ARG CRATE=""
 ARG TARGETOS TARGETARCH TARGETVARIANT
+RUN curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash -s -- \
+        --version ${WASMEDGE_VERSION} \
+        --platform $(xx-info os | sed -e 's/\b\(.\)/\u\1/g') \
+        --machine $(xx-info march) \
+        --path /usr/local
+
 RUN --mount=type=cache,target=/usr/local/cargo/git/db \
     --mount=type=cache,target=/usr/local/cargo/registry/cache \
     --mount=type=cache,target=/usr/local/cargo/registry/index \
     --mount=type=cache,target=/build/src/target,id=runwasi-cargo-build-cache-${TARGETOS}-${TARGETARCH}${TARGETVARIANT} <<EOT
     set -e
     export "CARGO_NET_GIT_FETCH_WITH_CLI=true"
+    export RUSTFLAGS='-Clink-arg=-Wl,-rpath,$ORIGIN'
```

Comment:
We are by default statically linking against wasmedge now. You can remove this.

Reply:
done.